### DISCUSSION TITLE
What do the P labels for issues stand for?

### DISCUSSION THREAD

**Discussion by azangru** [storybookjs/storybook#14490] [Documentation] [answered]
I've seen some issues which, when triaged, are marked with different P labels — there are P0 through P3, plus PN.

![image](https://user-images.githubusercontent.com/6834224/113712183-b8f5fe00-96dd-11eb-83b1-3d6db7d74c64.png)

I guess P stands for priority, but I couldn't find whether P0 is the most urgent one or the other way around. There's a [Maintainers.md](https://github.com/storybookjs/storybook/blob/next/MAINTAINERS.md) file with some labels explained, but it doesn't include the P ones. Could someone please clarify? If these are priority labels, it would be nice to know, even for non-contributors, how highly an issue is prioritized.

**Comment by jonniebigodes** [comment, marked as answer]
@azangru, you're right. The "P(x)" is related to the issue/bug priority. These were included recently, and I haven't gotten around to document them as I'm currently working on some other items! I've added it to my todos, and a pull request will be up shortly to address this and provide clarity.

Stay safe

**Reply by jonniebigodes** [reply]
@azangru a recent pull request was merged addressing this discussion. 

Thanks for bringing this to our attention. Let us know of any further issues you encounter.

Stay safe

**Reply by azangru** [reply]
Thank you for the update! 👍
